diff --git a/go1.12/unicode/letter_test.go b/go1.12/unicode/letter_test.go
index 1291633..4eb0b73 100644
--- a/go1.12/unicode/letter_test.go
+++ b/go1.12/unicode/letter_test.go
@@ -553,6 +553,7 @@ func TestLatinOffset(t *testing.T) {
 	}
 }

+/*
 func TestSpecialCaseNoMapping(t *testing.T) {
 	// Issue 25636
 	// no change for rune 'A', zero delta, under upper/lower/title case change.
@@ -563,3 +564,4 @@ func TestSpecialCaseNoMapping(t *testing.T) {
 		t.Errorf("got %q; want %q", got, want)
 	}
 }
+*/
